src/views/cross_reference.ts

```diff
@@ -40,8 +41,8 @@ export class CrossReferenceView<PropsType extends CrossReferenceViewProps>
     if (!rids.length) {
       return
     }
-    this.props.history.push({
-      pathname: this.props.history.location.pathname,
+    this.props.navigate({
```

Comment:
I think we were mentioning on the sprint planning call a possibility to use transactions instead of this. That is using [scrollIntoView](https://prosemirror.net/docs/ref/#state.Transaction.scrollIntoView). Have you considered that?

Reply:
We already using the scrollIntoView, check the following file: 
https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/9fd586176be060b3d9a255764681913a80e09d6d/src/useEditor.ts#L188

Reply:
Right, but I meant to use it directly without using history?